app/frontend/src/assets/scss/style.scss

```diff
@@ -17,6 +17,17 @@ p {
   line-height: 1.6;
 }
 
+a {
+  text-decoration: none;
+  &:hover {
+    text-decoration: underline;
+    button {
+      display: inline-block;
+      text-decoration: none;
+    }
+  }
```

Comment:
This doesn't appear to be underlining only the text portion of the button at all on either Firefox or Chrome - unless that is intended?

Reply:
This is intended, it's not supposed to underline in buttons (will confirm with Mina when I discuss some other UI stuff and the styling ticket all underline behaviour)